mappings/net/minecraft/entity/ai/pathing/PathNodeType.mapping

```diff
@@ -1,3 +1,5 @@
 CLASS net/minecraft/class_7 net/minecraft/entity/ai/pathing/PathNodeType
-	FIELD field_13 weight F
-	METHOD method_11 getWeight ()F
+	FIELD field_13 defaultPenalty F
```

Comment:
Should this be simply `penalty`, and we rename the mob entities' custom penalty values "overrides" instead

Reply:
I think `defaultPenalty` makes it clear that this is only the default penalty and that an entity-specific one can be assigned. Otherwise, someone looking at only the node type class might think the penalties are the same for all entities.